apps/cowswap-frontend/src/modules/trade/state/alternativeOrder/const.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Whether Edit Order feature is enabled
+ * Disabled for now until backend has the capacity to address issues raised on https://www.notion.so/cownation/Canceling-Replace-Orders-Edge-Case-5f1d0d38e3f541e7b454f8e7faa7fee1?pvs=4
+ */
+export const IS_EDIT_ORDER_ENABLED = false
```

Comment:
No edits until this flag is set to `true`.

Decided not to create a launch darkly feature flag because we might have additional work on UI depending on how the backend changes are implemented.